.github/actions/s3-upload/index.js

```diff
@@ -53,6 +47,42 @@ const uploads = await Promise.all(filesToUpload
       params.Key = `${dirPath}/${params.Key}`
     }
 
+    return params
+  })
+
+const existingFiles = await Promise.all(uploadCommandParams
+  .filter(params => params.CacheControl.indexOf('max-age=31536000') > 0)
```

Comment:
This looks good but just want to double check -- Is this sufficient to select the files that should be rewritten ( explicit loaders, chunks) but not those that need to allow it (.x loaders, current)

Reply:
The files that are not cached for one year are the same ones that should be overwritten with each release AND files that are not in the root of the S3 bucket.